meta-lmp-base/recipes-support/fioconfig/fioconfig_git.bb

```diff
@@ -31,6 +31,8 @@ do_install_append() {
 	install -d ${D}${systemd_system_unitdir}
 	install -m 0644 ${WORKDIR}/fioconfig.service ${D}${systemd_system_unitdir}/
 	install -m 0644 ${WORKDIR}/fioconfig-extract.service ${D}${systemd_system_unitdir}/
+	install -d ${D}${bindir}
+	install -m 0755 ${S}/src/${GO_IMPORT}/contrib/aktualizr-toml-update ${D}${bindir}
```

Comment:
this is shear curiosity - my version of this same change had 0655 for the file mode. I didn't find any clear guidance on what you should mark binaries with?

Reply:
As this is a script it should have +x by default, otherwise I'm surprised it worked.

Reply:
I believe it is probably better to have this script available via /usr/share or /usr/lib instead, in a folder that is specific to fioconfig, otherwise it could later create conflicts with other binaries, besides the confusion on the script scope.

Reply:
@doanac 
A value of `6` for `user` (vs. `5` for `group` and `other`) would be: `read + write`
Where a value of `7` for user would be: `read + write + execute`

You could in theory leave this as 0644 (u=read+write, go=read) and make sure to call it with:
`/bin/sh /usr/bin/aktualizr-toml-update`
But I don't think we want that as some handlers may not be shell scripts.

Reply:
It is fine for the script to have +x for every user.